site/_posts/2023-03-10-growing-is-hard.md

```diff
@@ -0,0 +1,63 @@
+
+One report shows an average tenure in big tech is between 1.1 and 3.2 years.[^1] The US Bureau of Labor and Statistics report a nationwide average of 3.8 to 4.3 years which is not limited to the software industry.[^2] Regardless of which statistic you use, it is hard to argue that average tenure is short, and almost certainly lower than the average for the general US population.
+
+### Bye, Felicia
+Switching jobs means new experiences, more money and a change of scenery. This can be very appealing early in your career and can help you level up quickly.
+
+The downside is that you rarely develop the same level of experiences and skill that you would by sticking around, learning to work through conflict, mastering a code base and learning how to stay productive in spite of company politics.
+
+Unfortunately this doesn't seem to be the sentiment shared by many engineers. 
+
+### The problem
+By the time an employee is beginning to produce and become competent - they have one foot out the door looking for their next adventure. 
+
+There are various ways to overcome this problem.
+
+Venture capital backed companies solve this by hiring many engineers at once. Using this strategy, even if a few leave, there is nearly always overlap in tenure so that the combined skills on the team never go to zero.
+
+At FusionAuth, we've taken a different path. things a little different. We've never taken capital, we've built a profitable business w/out debt. We hire when we can afford to do so.
+
+Hiring slow, means we may only be able to afford to hire a new engineer every 12-18 months. If the average tenure is less than your hiring cadence - you'll never get ahead of the attrition curve.
+
+It is sort of like filling a tub without plugging the drain. Unless you have great water pressure , the tub will drain faster than you can fill it. It is just difficult to get ahead, and you end up sitting cold and naked in the tub. The analogy does fall apart a bit at the end.   
+
+The result is that it leaves the technical founders holding the bag every time. This cycle repeats and can become demoralizing when the time and effort that is spent imparting the skills necessary to continue and grow the business are never preserved.
+
+### Light at the end of the tunnel
+
+Thankfully, we have turned the corner on this issue and begun to dig our way out of the whole. As FusionAuth can continued to grow, we can now afford to pay competitive salaries, and we also have enough revenue to hire in bulk. Hiring more than one engineer at a time is helpful in several ways. 
+
+The first is that the new team feels a sense of community as they all start from zero together. They do not have to compare their skill set or knowledge of the product with someone that has been doing it for 10 years, and instead can compare to another that is learning the same things.
```

Comment:
```suggestion
The first is that the new team feels a sense of camaraderie as they all start from "zero" together. They do not have to compare their skillset or knowledge of the product with someone that has been working in the space for 10 years, and instead can compare to another that is starting from the same place at the same time.
```

The "compare to another" also sounds like it's a competition. Not sure if there is a better way to phrase that.